src/main/java/io/gingersnapproject/metrics/micrometer/MicrometerMetrics.java

```diff
@@ -62,6 +62,8 @@ void registerMetrics(@Observes Events.ConnectorStartedEvent ev) {
          case MYSQL -> registerMysqlConnectorMetrics(rule);
          case SQLSERVER -> registerGenericConnector(io.debezium.connector.sqlserver.Module.name(), rule);
          case POSTGRESQL -> registerGenericConnector(io.debezium.connector.postgresql.Module.name(), rule);
+         // TODO -> oracle metrics available in OracleStreamingChangeEventSourceMetricsMXBean
```

Comment:
This PR or a follow up?

Reply:
Follow-up, tracked by https://github.com/gingersnap-project/db-syncer/issues/71